spec/workers/gitter_notificator_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'spec_helper'
+
+describe GitterNotificator do
+  let(:repository) { "owner/name" }
+  let(:branch) { "master" }
+  let(:room) { "owner/name" }
+
+  describe "#init" do
+    let(:token) { "token" }
+    let(:item_branch) { double("Branch", :repo => "R", :branch => "B", :gitter_room => "GR") }
+    let(:item_travis_monitor) { double("travis_monitor", :gitter_token => token) }
+
+    it "should initialize @gitter_token" do
+      allow(item_travis_monitor).to receive(:branches).and_return([])
+      allow(Settings).to receive(:travis_monitor).and_return(item_travis_monitor)
+
+      subject.send(:init)
+
+      expect(subject.instance_variable_get(:@gitter_token)).to eq(token)
+    end
+
+    it "should call check with proper parameters" do
+      allow(item_travis_monitor).to receive(:branches).and_return([item_branch])
+      allow(Settings).to receive(:travis_monitor).and_return(item_travis_monitor)
```

Comment:
I see similar mocks in multiple `it`s, can you please pull them out into a `before` block?